internal/v1/handler.go

```diff
@@ -57,7 +57,7 @@ func (h *Handlers) GetVersion(ctx echo.Context) error {
 }
 
 func (h *Handlers) GetReadiness(ctx echo.Context) error {
-	resp, err := h.server.cClient.OpenAPI(ctx.Request().Context())
+	resp, err := h.server.cClient.GetOpenapi(ctx.Request().Context())
```

Comment:
And the intermediate client is gone, all generated methods can be used directly.